code/datums/components/knockback.dm

```diff
@@ -1,22 +1,45 @@
 /datum/component/knockback
 	var/throw_distance
+	var/reverse
```

Comment:
Just have a negative throw distance?

Reply:
oh duh haha i got stuck up in the fact that getting negative target turfs wouldnt work